### ISSUE TITLE
docs: Issue on page Encryption: Data in Transit Client to Node

### ISSUE THREAD

**Issue by scylladb-buff** [scylladb/scylladb#19888]
I would like to report an issue on page https://opensource.docs.scylladb.com/branch-6.0/operating-scylla/security/client-node-encryption

### Problem
This page describes client-to-node encryption and how to configure the client and the node for 6.0. 
The node sections seem fine. 
But in the client section, we are focused on how to configure the Cassandra stress test and Java key store to work with SSL. 
Casandra-stress test is one of many clients that can use SSL, it is not even the most popular. 

###  Suggest a fix
You should add more frequently used clients:
cqlsh
phyton
golang

**Comment by tzach** [comment]
An alternative would be to point to the relevant section in each driver doc (or both)
@DanielHe4rt can you please take a look?

**Comment by DanielHe4rt** [comment]
I've been working in something related TLS in these last days and just managed to make it work. 

If I understood corrently, this is a step ahead the Connection Encryption, right? Because it seems almost identical the normal Connection Encryption. Maybe I'm missing something??

Maybe we can sync (@scylladb-buff) so you can explain about this specific feature.

And btw, here's the TLS draft repo: https://github.com/DanielHe4rt/scylladb-role-tls-auth

**Comment by scylladb-buff** [comment]
I agree with @Tzach. We should make the client section more generic and link to code examples from it.

The documentation should show how our users can connect to a ScyllaDB using a TLS connection using our supported clients.
Right now, we cover one client, Casandra-stress test (Java)
